Gateway/Response/PaymentCaptureDetailsHandler.php

```diff
@@ -41,6 +41,9 @@ public function handle(array $handlingSubject, array $response)
         // set pspReference as lastTransId only!
         $payment->setLastTransId($response['pspReference']);
 
+        if($response["response"]=='[capture-received]'){
```

Comment:
```suggestion
        // The capture request will return a capture-received message, but it doesn't mean the capture has been final
        // so the invoice is set to Pending
        if($response["response"]=='[capture-received]'){
```

Reply:
Is there a reason why strict comparison is not used?